### DISCUSSION TITLE
PLEX Container with portable config

### DISCUSSION THREAD

**Discussion by flinkazoid** [tteck/Proxmox#438] [Help Needed]
Please forgive me for asking a potentially ignorant question.  Im use to using docker containers where I can pass through almost anyting I want into a mountpoint/folder existing or not.  

What Im trying to do here is use a pre-existing PMS configuration that is located on a network share and pass it into "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/"

The problem Im having is that no matter what I do the Server always boots up to a clean/new config.  

Is this possible with LXC?  Is this an issue with the priviliged container?  Has anyone else made a portable container like this?

**Comment by tteck** [comment]
Don't really understand what you're trying to do but, Google found this https://www.closingtags.com/writing-to-bind-mounts-from-unprivileged-lxc-containers/ maybe this will help 🤷🏻‍♂️

**Comment by thaidax** [comment]
> var/lib/plexmediaserver/Library/Application Support/Plex Media Server/

Hi 

> Please forgive me for asking a potentially ignorant question. Im use to using docker containers where I can pass through almost anyting I want into a mountpoint/folder existing or not.
> 
> What Im trying to do here is use a pre-existing PMS configuration that is located on a network share and pass it into "/var/lib/plexmediaserver/Library/Application Support/Plex Media Server/"
> 
> The problem Im having is that no matter what I do the Server always boots up to a clean/new config.
> 
> Is this possible with LXC? Is this an issue with the priviliged container? Has anyone else made a portable container like this?

I am trying to do the same thing with no success, did you find a solution in the end?
